src/ontology/ro-edit.owl

```diff
@@ -5646,6 +5648,18 @@ AnnotationAssertion(oboInOwl:creation_date obo:RO_0004047 "2018-03-13T23:55:19Z"
 AnnotationAssertion(rdfs:label obo:RO_0004047 "causally upstream of or within, positive effect")
 SubObjectPropertyOf(obo:RO_0004047 obo:RO_0002418)
 
+# Object Property: obo:RO_0007000 (has_driver)
+
+AnnotationAssertion(obo:IAO_0000115 obo:RO_0007000 "A relation between a material entity and a system, in which the material entity is any natural or human-influenced factor that directly or indirectly causes a change in the system.")
```

Comment:
The definition should be looked at by @cmungall @nicolevasilevsky @balhoff or anyone with a content focus

Reply:
this def looks good to me.

Reply:
When devising the 'has driver' definition, I was looking to encompass the broader idea of a driver of change in systems, with disease being only one of those systems. Some references: 
https://www.greenfacts.org/glossary/def/driver.htm

https://www.who.int/docs/default-source/climate-change/qa-infectiousdiseases-who.pdf?sfvrsn=3a624917_3

Cheers,
Lynn

Reply:
The text definition implicitly states domain and range:

 - domain: material entity
 - range: system

In general, domain and range constraints should be explicitly stated as logical axioms. But before doing that, is this really what you mean? You would not be able to use this with diseases if diseases are modeled as BFO dispositions.

We will also want to work on the range constraint. There is a class "system" in RO that was added for ENVO but has faced some scrutiny, and moving forward we want domains and ranges in RO to be in COB.

My suggestion is to weaken the implicit domain and range constraints and say in the text "between two entities" and add a comment saying that domain/ranges are being worked on pointing back to the ticket in github. We can then merge this and iterate on improvements

Or if you prefer we could try and get it all right in one PR - this may take longer though.

Reply:
'between two entities' is a good solution. 
I will update the file. 
Thank you Chris !! 
Cheers,
Lynn